ZupIT/ritchie-cli
# Kill bill(d)

### Description

#578 has introduced significant progress for formula builds in terms of UX. We have opted to eliminate the concept of building completely for the user. For that we need to make some adjustments to the code:

* Deprecate the `build formula` command
* Remove the formula watcher, since it was only used with the build
* Remove the watcher lib
* Remove the auto-rebuild prompt on the formula pre-runner. We should always rebuild it
* Run the build when adding a new workspace
* Fix tests accordingly

### Observations

The auto build was not working since it checks for a repo named `local`. However, we have refactored the code to break local workspaces into multiple repos, rendering the autobuild useless

By setting up the workspace during the `add` action. We now have access to all formulas and their autocomplete whenever adding a new workspace. There is no need to call an explicit `build` for the formulas to be added

### How to verify it
Ritchie should function normally and without the need to use the `rit build formula` command

### Changelog

Deprecated build formula command, auto-rebuild is the main choice now
